bophono/data/exceptions-kvp.csv

```diff
@@ -31,6 +44,7 @@
 ས་གདན,sabd-en
 ཁ་གཅོད,khabch-ö
 # dba becomes wa
+དབ,w-a
 དབ/Cb,w-
```

Comment:
please add tests for 
- `'a`
- `'od`
- `dba`
- `dbo`
- `dbos`
- `dbu`
- `dbus`
- `dbi`
- `dbe`